scripts/rollup/forks.js

```diff
@@ -34,7 +34,11 @@ const forks = Object.freeze({
       // happens. Other bundles just require('object-assign') anyway.
       return null;
     }
-    if (entry === 'react') {
+    if (
+      entry === 'react' ||
```

Comment:
Let's make it `entry === 'react' || entry.startsWith('react/')` so that we don't forget in case of a rename.